linchpin/provision/roles/openshift/tasks/provision_resource_group.yml

```diff
@@ -3,20 +3,16 @@
   set_fact:
     auth_var: ""
 
-- name: "set default_cred_profile when res_grp[credentials] is undefined"
-  set_fact:
-    cred_profile: 'default'
-  when: res_grp['credentials'] is not defined
-
 - name: "set default_cred_filename when res_grp[credentials] is undefined"
   set_fact:
     cred_filename: 'openshift.yaml'
   when: res_grp['credentials'] is not defined
 
+  # sets cred profile as default if not found if not found
```

Comment:
nit: double, and maybe we need to put what if not mentioned/found, what do you think?